addons/binding/org.openhab.binding.dscalarm/ESH-INF/thing/panel.xml

```diff
@@ -0,0 +1,194 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dscalarm"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+     <thing-type id="panel">
+
+	    <supported-bridge-type-refs>
+            <bridge-type-ref id="envisalinkbridge" />
+            <bridge-type-ref id="it100bridge" />
+        </supported-bridge-type-refs>
+
+        <label>DSC Alarm Panel</label>
+        <description>The basic representation of the DSC Alarm System.</description>
+        
+        <channels>
+            <channel id="panel_message" typeId="panel_message"/>
+            <channel id="panel_command" typeId="panel_command"/>
+            <channel id="panel_system_error" typeId="panel_system_error"/>
+            <channel id="panel_trouble_message" typeId="panel_trouble_message"/>
+            <channel id="panel_trouble_led" typeId="panel_trouble_led"/>
+            <channel id="panel_service_required" typeId="panel_service_required"/>
+            <channel id="panel_ac_trouble" typeId="panel_ac_trouble"/>
+            <channel id="panel_telephone_trouble" typeId="panel_telephone_trouble"/>
+            <channel id="panel_ftc_trouble" typeId="panel_ftc_trouble"/>
+            <channel id="panel_zone_fault" typeId="panel_zone_fault"/>
+            <channel id="panel_zone_tamper" typeId="panel_zone_tamper"/>
+            <channel id="panel_zone_low_battery" typeId="panel_zone_low_battery"/>
+            <channel id="panel_time_loss" typeId="panel_time_loss"/>
+        	<channel id="panel_time" typeId="panel_time"/>
+            <channel id="panel_time_stamp" typeId="panel_time_stamp"/>
+            <channel id="panel_time_broadcast" typeId="panel_time_broadcast"/>
+            <channel id="panel_fire_key_alarm" typeId="panel_fire_key_alarm"/>
+            <channel id="panel_panic_key_alarm" typeId="panel_panic_key_alarm"/>
+            <channel id="panel_aux_key_alarm" typeId="panel_aux_key_alarm"/>
+            <channel id="panel_aux_input_alarm" typeId="panel_aux_input_alarm"/>
+        </channels>
+
+		<config-description>
+
+			<parameter name="userCode" type="text" required="false">
+               <label>User Code</label>
+                <description>The User Code.</description>
+                <default>1234</default>
+            </parameter>
+
+			<parameter name="suppressAcknowledgementMsgs" type="boolean" required="false">
+               <label>Suppress Acknowledgement Messages</label>
+                <description>Suppress Acknowledgement Messages When Received.</description>
+                <default>false</default>
+            </parameter>
+
+		</config-description>
+
+    </thing-type>
+
+    <channel-type id="panel_message">
```

Comment:
Are any of the channel types something that you would consider to not be the "main functionality" and thus could be marked as "advanced" (which will make it appear in the UI only on user request)?